### PR TITLE
BLADE-257 Recognize commands with spaces

### PR DESCRIPTION
**Pull request by christopherbryanboyd**
Hi @gamerson , this addresses https://issues.liferay.com/browse/BLADE-257

### CODE REVIEW

File: cli/src/test/java/com/liferay/blade/cli/command/ServerStartCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/**
```
Comment (gamerson) [inline cli/src/test/java/com/liferay/blade/cli/command/ServerStartCommandTest.java]: Can you collapse both of these two tests into one class called `ServerCommandsTest` and then have two methods in there, testing `server start` and `server stop`?

File: cli/src/test/java/com/liferay/blade/cli/TestUtil.java [hunk 2]
```diff
@@ -63,6 +64,21 @@ public FileVisitResult visitFile(Path path, BasicFileAttributes basicFileAttribu
 			});
 	}
 
+	public static boolean doesCommandExist(String... args) {
```
Comment (gamerson) [inline cli/src/test/java/com/liferay/blade/cli/TestUtil.java]: Can you move this static method to a private static method on the new `ServerCommandsTest`?

**Comment by christopherbryanboyd** [comment]
ci:test

**Comment by christopherbryanboyd** [comment]
ci:test

**Comment by christopherbryanboyd** [comment]
ci:test

**Comment by christopherbryanboyd** [comment]
ci:test

**Comment by christopherbryanboyd** [comment]
ci:test

**Comment by christopherbryanboyd** [comment]
ci:test

**Comment by christopherbryanboyd** [comment]
Hi @gamerson , updated PR submitted here https://github.com/gamerson/liferay-blade-cli/pull/460 as CI refused to test this one any further

**Pull request closed without merging**